xchange-bitstamp/src/main/java/org/knowm/xchange/bitstamp/service/BitstampDigestV2.java

```diff
@@ -0,0 +1,76 @@
+package org.knowm.xchange.bitstamp.service;
+
+import org.knowm.xchange.service.BaseParamsDigest;
+import si.mazi.rescu.RestInvocation;
+
+import javax.crypto.Mac;
+import javax.ws.rs.HeaderParam;
+import java.math.BigInteger;
+
+/** @author Benedikt Bünz */
+public class BitstampDigestV2 extends BaseParamsDigest {
+
+  private final String apiKey;
+  private final String baseUrlHost = "www.bitstamp.net";
+
+  /**
+   * Constructor
+   *
+   * @param secretKeyBase64
+   * @param apiKey @throws IllegalArgumentException if key is invalid (cannot be base-64-decoded or
+   *     the decoded key is invalid).
+   */
+  private BitstampDigestV2(String secretKeyBase64, String apiKey) {
+
+    super(secretKeyBase64, HMAC_SHA_256);
+    this.apiKey = "BITSTAMP "+ apiKey;
+  }
+
+  public static BitstampDigestV2 createInstance(
+      String secretKeyBase64, String apiKey) {
+
+    return secretKeyBase64 == null ? null : new BitstampDigestV2(secretKeyBase64, apiKey);
+  }
+
+  @Override
+  public String digestParams(RestInvocation restInvocation) {
+
+    Mac mac256 = getMac();
+    mac256.update(apiKey.getBytes());
+
+    System.out.println("restInvocation.getHttpMethod() " + restInvocation.getHttpMethod());
```

Comment:
please remove all these System.out.println

Reply:
Oups sorry forgot about that :)
Will do now ...